src/test/resources/application.yml

```
@@ -0,0 +1,21 @@
+    driver-class-name: org.h2.Driver
+    username: sa
+    password:
+
+jwt:
+  secret:
+    key: VEVTVF9KV1RfU0VDUkVUX0tFWV9URVNUX0pXVF9TRUNSRVRfS0VZX1RFU1RfSldUX1NFQ1JFVF9LRVk= # 테스트용 시크릿 키
+  access:
+    expiration: 1800000 # 30분
+  refresh:
+    expiration: 604800000 # 7일
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)